### ISSUE TITLE
dials tests need dials_data ?

### ISSUE THREAD

**Issue by asmit3** [dials/dials#887]
While trying to run ```libtbx.run_test_parallel module=dials```, I kept getting several test failures related to dials_data. Is this a new repository for the dials builder or am I missing something ? My build is up to date. See below for one of the representative test failures.

```
    file /net/dials/raid1/asmit/ExaFEL/dials/modules/dials/algorithms/scaling/test_scale.py, line 365
      def test_scale_and_filter_dataset_mode(dials_data, tmpdir):
    E       fixture 'dials_data' not found
    >       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, dials_regression, doctest_namespace, monkeypatch, pytestconfig, record_property, record_xml_attribute, recwarn, run_in_tmpdir, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
    >       use 'pytest --fixtures [testpath]' for help on them.
```

**Comment by graeme-winter** [comment]
The test in question should probably test for `dials_data` fixture before trying to use it... I think that is a standard motif in other tests. Also, for dials developers `libtbx.pip install -U dials_data` is useful to do. 

@asmit3 `dials_data` is a pip installable public metadata archive which points to where example test data are, to avoid the need for keeping everything in the `dials_regression` private SVN repo (which has two problems - it's private and SVN so keeps every file at least twice).

**Comment by graeme-winter** [comment]
Assigned @jbeilstenedmands to clean up the test to check for the fixture before using it

**Comment by Anthchirp** [comment]
dials_data has been a declared dependency for 4 months. Please run `libtbx.show_python_dependencies` to check your dependencies.

**Comment by jbeilstenedmands** [comment]
I've checked the test and the format is correct - the individual tests do not need to check for the fixture. The scaling tests are the first ones picked up in dials, so I imagine this is why these are the ones showing first, but I don't think the tests themselves are the issue.

**Comment by ndevenish** [comment]
Did you run `libtbx.refresh` when you updated?

**Comment by asmit3** [comment]
@graeme-winter @Anthchirp I wasn't aware this is now a required dependency for dials testing. I followed the instruction in your comments and installed dials_data and the tests pass now. Thanks for addressing this issue. 
@ndevenish I didn't run ```libtbx.refresh``` but I did run ```python bootstrap.py hot update build --builder=dials``` to update my sources. My guess is that should run libtbx.refresh ? Anyhow the issue has been fixed so happy to close this.
